.idea/codeStyleSettings.xml

```diff
@@ -1,264 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why were these files removed? They are in the repository on purpose.

Reply:
I did not actively remove them. No idea how that happened. I’ll make sure they are there again when I make the pull request

Reply:
That's usually why you check what files you are commiting, before doing the commit :clinking_glasses:

Reply:
@thellmund please revert changes to .idea files

Reply:
I know I forgot something... added them again.